pages/registrations.py

```diff
@@ -0,0 +1,20 @@
+import settings
+
+from selenium.webdriver.common.by import By
+from base.locators import Locator, GroupLocator
+from pages.base import OSFBasePage
+
+
+class MyRegistrationsPage(OSFBasePage):
+    url = settings.OSF_HOME + '/registries/my-registrations/'
+    identity = Locator(By.CSS_SELECTOR, 'div[data-analytics-scope="Registries"]')
+
+    drafts_tab = Locator(By.XPATH, '//a[text()="Drafts"]')
```

Comment:
would be nice to avoid depending on localized display text

Reply:
Agreed. The FE Team has added new data test locators for me to access the tabs!

- `data-test-my-registrations-nav="drafts"`
- `data-test-my-registrations-nav="submitted"`